src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -445,4 +445,12 @@ default boolean damage(double damage, DamageSource damageSource) {
      * @param uuid The {@link UUID} to set as notifier.
      */
     void setNotifier(@Nullable UUID uuid);
+
+    /**
+     * Returns whether this entity can see the provided {@link Entity}.
+     *
+     * @param entity The entity to check visibility for
+     * @return {@code true} if this entity can see the provided entity
+     */
+    boolean canSee(Entity entity);
```

Comment:
I mean, is this just going to delegate to the regular ContextualData API? If so, it should probably be a default method.